assets/state-driver/0500_daemonset.yaml

```diff
@@ -88,6 +88,25 @@ spec:
             - name: run-mellanox-drivers
               mountPath: /run/mellanox/drivers
               mountPropagation: HostToContainer
+        - name: check-imex-config
```

Comment:
Why does this have to be a separate sidecar container?

Can we not add this logic to the main driver container scripts?